### ISSUE TITLE
Not recognizing some atlantis relics

### ISSUE THREAD

**Issue by markotuna** [Malgardian/TQCollector#19]
Epic Coral Fragment, Epic Atlantean Artifice (quest relic) and Legendary Atlantean Artifice arent recognized as obtained when you have them.
![image](https://user-images.githubusercontent.com/82469901/114602268-b3c52000-9c96-11eb-993b-b85c5b06d07c.png)
![image](https://user-images.githubusercontent.com/82469901/114602302-be7fb500-9c96-11eb-9bf6-2891aafaa3ce.png)

**Comment by DarthSomebody** [comment]
Atlantean Artifice still isn't recognized, but coral fragments are.

![image](https://user-images.githubusercontent.com/10331302/183204210-83a9dfdf-6149-44c7-898e-05e87f43d9e5.png)

**Comment by Malgardian** [comment]
@DarthSomebody Did you acquire one recently at a current patch state of the game?

**Comment by Davb37** [comment]
TQCollector v1.4.0.1
TQVaultAE 4.3.0
Titan Quest - Anniversary Edition v2.10.21415
Atlantean Artifice still not recognised:
![1](https://user-images.githubusercontent.com/121893116/212492393-1a239602-d83a-4ec2-b5d2-9ec33432d50f.jpg)
![2](https://user-images.githubusercontent.com/121893116/212492398-a060dcb1-4e37-4715-8faa-acbfc8f93ec5.jpg)

**Comment by Malgardian** [comment]
@Davb37 Thanks for the report. Could you put a copy of that charm into an empty vault file and attach it here? Thanks.

**Comment by Davb37** [comment]
Here it is:
[New Vault.vault.json.zip](https://github.com/Malgardian/TQCollector/files/10418998/New.Vault.vault.json.zip)

**Comment by Malgardian** [comment]
I see the issue now, the game is still incorrectly creating that charm. I'll need to think about how to fix this properly.

For the time being you can just open the itemdb.xml file of TQC and replace the "\"s with "/"s for the entry of the Atlantean Artifice charm.

**Comment by ezloj** [comment]
How should this be modified? @Malgardian 
`
        <Item dbr="xpack3\items\charms\01_x3_quest_artifice.dbr" id="x3tag_animalpart04" isIT="true" isR="false" isAtl="true" isEE="false" find=""/>`
        
        There is only one `""` and adding a slash in it does nohing

**Comment by Davb37** [comment]
You need to change all the \ to /, like that:
xpack3/items/charms/01_x3_quest_artifice.dbr
You need to do it for the 3 lines: normal, epic and legendary

**Comment by ezloj** [comment]
That did it. Thank you.

**Comment by Malgardian** [comment]
@ezloj Yeah, unfortunately Github's markdown swallowed my lone "\\". Sorry about the confusion. Eventually I'll have a proper fix, but time's in short supply currently.

**Comment by ezloj** [comment]
It looks like the basilisk eye is still not recognised. Not trying to rush here, just saying. The fix with inverted slash doesn't help in this case.
